waspc/examples/todoApp/src/client/App.tsx

```diff
@@ -1,18 +1,25 @@
 import { Link } from 'react-router-dom'
 
 import logout from '@wasp/auth/logout.js'
-import useAuth from '@wasp/auth/useAuth.js'
+import useAuth from '@wasp/auth/useAuth'
+import { useQuery } from '@wasp/queries'
+import getDate from '@wasp/queries/getDate'
 
 import './Main.css'
-export function App({ children }) {
+
+export function App({ children }: any) {
   const { data: user } = useAuth()
+  const { data: date } = useQuery(getDate)
 
   return (
     <div className="app border-spacing-2 p-4">
       <header className="flex justify-between">
         <h1 className="font-bold text-3xl mb-5">
           <Link to="/">ToDo App</Link>
         </h1>
+        <h2>
+          Your site was loaded at: {date?.toLocaleString()}
```

Comment:
Like in the good old PHP websites. 
![Screenshot 2023-04-06 at 16 55 28](https://user-images.githubusercontent.com/2223680/230417908-14bfe064-e232-4e68-a3e0-f9e55a79e11f.png)